tests/test_math/test_aggregate.py

```diff
@@ -4,26 +4,53 @@
 import nujo.math.aggregate as aggregate
 from nujo.init.random import rand
 
+# ====================================================================================================
+# Test Summation
+
 
 def test_sum(inputs):
-    assert (aggregate.sum(inputs[0]) == sum(inputs[0].value)).all()
+    output = aggregate.sum(inputs[0])
```

Comment:
Why make a new variable when you only use it once

Reply:
Usually, I would not do that, you are right,  however here I did it for two reasons:
 1) Improve readability.
 2) Make it consistent with `test_prod` where `output` is used twice

Reply:
I see, LGTM :)